config/v1/types_infrastructure.go

```diff
@@ -1519,7 +1519,14 @@ type IBMCloudServiceEndpoint struct {
 
 // IBMCloudPlatformSpec holds the desired state of the IBMCloud infrastructure provider.
 // This only includes fields that can be modified in the cluster.
-type IBMCloudPlatformSpec struct{}
+type IBMCloudPlatformSpec struct {
+	// ServiceEndpoints contains custom endpoints designated to
+	// override existing defaults of IBM Cloud Services.
```

Comment:
The godoc on the status version of this field is different, shall we copy it here?

Reply:
yes, fixed